css/team-carousel.css

```diff
@@ -0,0 +1,248 @@
+/* Team Carousel - Final Version */
+.team-carousel-container {
+  position: relative;
+  max-width: 1200px;
+  margin: 0 auto;
+  padding: 0 50px;
```

Comment:
```suggestion
  padding: 0 0;
```
Would look better. as nor picture is cut off in the middle of the page. We could add something like these dots to make it look clean 
![image](https://github.com/user-attachments/assets/8b5735d7-21a1-468c-b879-82a326dc7c7b)

Reply:
This is the way i created the dots for the 3d model we could use the same stile here 

````js
pages.forEach((page, i) => {
      const dot = document.createElement('button');
      dot.className =
        'h-2 w-2 rounded-full bg-gray-300 transition-colors';
      dot.onclick = () =>
        page.scrollIntoView({behavior:'smooth', inline:'center'});
      nav.appendChild(dot);
    });
````